### DISCUSSION TITLE
Why are trailing spaces in "public search" deleted while typing?

### DISCUSSION THREAD

**Discussion by RealMurphy** [LycheeOrg/Lychee#2045] [Q&A] [answered]
Hi, just set up my first instance and noticed that when searching for images, trailing spaces are automatically deleted while typing, i.e. typing `this and that` results in `thisandthat` and one does not find images form the tag `this and that` anymore.

One can add the spaces after the fact and the search then provides the correct images, but it feels a bit weird doing it that way.

Anyone knows why it was implemented like that?

**Comment by qwerty287** [comment]
This looks more like a bug. Probably it's fixed in the new livewire frontend already.

**Reply by RealMurphy** [reply]
I see. Given that I never really tinkered with PHP, that means just wait for the next Lychee release and upgrade then?

**Reply by qwerty287** [reply, marked as answer]
Yes, but there's no release date yet, we need to see how long it takes.

**Reply by RealMurphy** [reply]
good enough for me, thanks!
